### DISCUSSION TITLE
Is it true that public Reddit zims are illegal because of copyright?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [openzim/zim-requests#1077] [Q&A] [answered]
Only local zim creating is ok?

**Comment by Jaifroid** [comment]
All I know is that reddit now monetize their API access and limit or ban web scrapers. There is no open access to their API any more, you have to pay them for access. This was the cause of the big rebellion earlier this year or late last year, where unpaid moderators stopped moderating or locked forums in protest.

**Comment by Popolechien** [comment, marked as answer]
We did contact them a while back and their legal team explicitely said no. Creating your own personal copy (via zimit for instance) is more subject to local laws (in Switzerland for instance it is probably legal).
